api/app/views/mno_enterprise/jpi/v1/organizations/_current_user.json.jbuilder

```diff
@@ -2,4 +2,4 @@ json.id user.id
 json.name user.name
 json.surname user.surname
 json.email user.email
-json.role organization.members.to_a.find { |e| e.id == user.id }.role
+json.role user.role(organization)
```

Comment:
There is no performance issue, as the user is already loaded with it's organization.